pages/1.11/installing/production/system-requirements/ports/index.md

```diff
@@ -1,40 +1,44 @@
 ---
 layout: layout.pug
 title: DC/OS Ports
-menuWeight: 5
+navigationTitle: Ports
+menuWeight: 20
 excerpt: Making sure ports are available for installation
-
-enterprise: false
 ---
 
 [DC/OS components](/docs/1.11/overview/architecture/components/) listen on multiple ports on each node. These ports must be available for installation to succeed.
 
 DC/OS allocates additional ports to services running on top of DC/OS. These ports are required to be available when services are installed.
 
-**Important:** These ports must not be used in a firewall configuration between nodes or cluster zones. For DC/OS to install and function as intended, these ports are open and accessible upon initial installation. Therefore, network-specific security measures - from outside of the cluster as well as between internal cluster nodes and zones - for each of these ports should be evaluated and, if necessary, put in place by the network administrator before installing and implementing DC/OS. Moreover, DC/OS security modes ("disabled", "permissive", and "strict") do not affect access to these ports.
+**Important:** 
+- These ports must not be used in a firewall configuration between nodes or cluster zones.
+- For DC/OS to install and function as intended, these ports are open and accessible upon initial installation. 
+- Therefore, network-specific security measures - from outside of the cluster as well as between internal cluster nodes and zones - for each of these ports should be evaluated and, if necessary, put in place by the network administrator before installing and implementing DC/OS. Moreover, DC/OS security modes ("disabled", "permissive", and "strict") do not affect access to these ports.
 
 ## All nodes
 
 ### TCP
 
 | Port | DC/OS Component | systemd Unit | Source | Destination |
 |---|---|---|---|---|
+| 53    | DC/OS Net | `dcos-net.service` | agent/master | agent/master [oss type="inline" size="small" /] | 
 | 61003 | REX-Ray | `dcos-rexray.service` | agent/master (may change due to specific REX-Ray configuration)| agent/master (may change due to specific REX-Ray configuration) |
-| 61053 | Mesos DNS | `dcos-mesos-dns.service` | agent/master | master |
+| 61053 | Mesos DNS | `dcos-mesos-dns.service` | agent/master | master [enterprise type="inline" size="small" /] | 
```

Comment:
mesos dns uses udp and tcp ports 61053 only on master nodes. it's not enterprise only.